### PR TITLE
Use Python 3.8 in pytest

### PR DESCRIPTION
**Pull request by abidwael**

### CODE REVIEW

**Review by arnavgarg1 — COMMENTED** [review]
Should we make this py3.8, py3.9 and py3.10?

**Comment by abidwael** [comment]
Currently, we only have tests on Ray 2.2.0 and 2.3.1 and we'll add the nightly tests back. We can do 3.8 and 3.10 for now. WDYT?

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -22,18 +22,21 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.8", "3.8"]
```
Comment (arnavgarg1) [inline .github/workflows/pytest.yml]: I think this needs to be 3.8, 3.9 and 3.10?

**Review by arnavgarg1 — APPROVED** [review]

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (abidwael) [reply .github/workflows/pytest.yml]: yeah just pushed a change

Comment (arnavgarg1) [reply .github/workflows/pytest.yml]: perfect, approved

**Pull request merged**